bin/templates/cordova/lib/builders/GradleBuilder.js

```diff
@@ -203,6 +203,11 @@ GradleBuilder.prototype.build = function(opts) {
         } else {
             process.stdout.write(stdio.stdout);
         }
+    }).catch(function (error) {
+        if (error.toString().indexOf('failed to find target with hash string') >= 0) {
+            return check_reqs.check_android_target(error);
```

Comment:
Would it be possible for check_android_target to actually find the target and then swallow this error?